### ISSUE TITLE
Support for "X-Robots-Tag" header

### ISSUE THREAD

**Issue by JamoCA** [atuttle/Taffy#417]
I noticed that WordPress's API added support for the [`X-Robots_tag`](https://ahrefs.com/blog/meta-robots/#what-is-x-robots-tag) header about 5 years ago and wonder if it's something that should be added as a default Taffy API response header.

- https://core.trac.wordpress.org/ticket/36390
- https://support.google.com/webmasters/thread/18632399/how-to-stop-google-to-access-internal-api-urls-which-are-part-of-inline-js?hl=en

While this header can already be manually added as a custom header in the Application.cfc, I'm not aware of any reasons why this wouldn't be beneficial to add as a default.

```
globalHeaders = {
	"X-Robots-Tag" = "noindex, nofollow, noimageindex"
}
```

Thoughts?

**Comment by atuttle** [comment]
I agree that it seems harmless enough, but I wouldn't want to add it without some way to remove it if someone decides they don't want it.
